src/consensus/lib.zig

```diff
@@ -1,5 +1,13 @@
 pub const fork_choice = @import("fork_choice.zig");
+pub const progress_map = @import("progress_map.zig");
+
+comptime {
+    _ = fork_choice;
+    _ = progress_map;
+}
```

Comment:
What is this for?

Reply:
Removed, I forget why it was there in the first place, I think I was running into an issue in an earlier iteration and this worked around it.